### PR TITLE
Emojis!

### LINKED ISSUE #1616: Support emoji in labels
GitHub [recently released](https://blog.github.com/2018-02-22-label-improvements-emoji-descriptions-and-more/) a feature allowing you to add emoji to label names. This allows you to do `bug 🐛` instead of the boring `bug`.

GItHawk doesn’t yet recognize emoji in labels and just renders it as a string. It looks like this:
a914c35c-6302-4604-9bd7-e23261202107

### PR DESCRIPTION
**Pull request by Huddie**
:bug: :smile:
Yay! Closes #1616

Simulator Screen Shot - iPhone XR - 2019-03-11 at 14 33 24

### CODE REVIEW

**Comment by Huddie** [comment]
Note: We already have a method that converts `:bug:` to :bug: as well as a test. So I do not include a test in this PR

**Comment by Sherlouk** [comment]
Well that is simpler than I expected 😂

**Comment by Huddie** [comment]
@Sherlouk agreed! 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: Classes/Models/RepositoryLabel.swift [hunk 1]
```diff
@@ -12,11 +12,11 @@ import IGListKit
 final class RepositoryLabel: ListDiffable, Hashable, Equatable, ListSwiftDiffable {
 
     let color: String
-    let name: String
+    var name: String = ""
```
Comment (BasThomas) [inline Classes/Models/RepositoryLabel.swift]: Any reason you changed this?

Comment (Huddie) [reply Classes/Models/RepositoryLabel.swift]: If let you can’t mutate. Meaning I can’t switch `:bug:` no?

Comment (BasThomas) [reply Classes/Models/RepositoryLabel.swift]: You're only setting it once in the initializer though, isn't it?

Comment (Huddie) [reply Classes/Models/RepositoryLabel.swift]: Oh yes! I originally had a separate fix (I created my
Own method that did the mapping. And I needed `self` but that doesn’t apply anymore. I’ll PR a quick fix to this soon. Thank you for pointing that out

**Pull request merged**